### PR TITLE
feat(677): Add command validate endpoint

### PR DESCRIPTION
**Pull request by kumada626**
## Context
Like `template`, we need an endpoint that can validate command. So I added the command validate API endpoint. 

## Objective
Like `POST validator/template` endpoint,  I added an endpoint that validates commands.

## References
#677

### CODE REVIEW

File: test/plugins/validator/command.test.js [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+                        },
+                        message: '"version" is required',
+                        path: ['version'],
+                        type: 'any.required'
+                    }
+                ]);
+            })
+        );
+
+        it('returns BAD REQUEST for template that cannot be parsed', () =>
```
Comment (wdstar) [inline test/plugins/validator/command.test.js]: `template` still remains.

Comment (kumada626) [reply test/plugins/validator/command.test.js]: Oh, I forgot to fix it. Thanks.

File: plugins/command-validator.js [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+const validator = require('screwdriver-command-validator');
+
+/**
+ * Hapi Command Validator Plugin
+ *  - Validates sd-command.yaml and returns the parsed command with any
+ *    errors associated with it
+ * @method register
+ * @param  {Hapi.Server}    server
+ * @param  {Object}         options
+ * @param  {Function} next
```
Comment (ScottyFillups) [inline plugins/command-validator.js]: Small whitespacing inconsistency

Comment (kumada626) [reply plugins/command-validator.js]: Thanks, I added whitespaces.

**Pull request merged**